clarity/Cargo.toml

```diff
@@ -15,7 +15,7 @@ resolver = "2"
 
 [lib]
 name = "clarity"
-path = "./src/libclarity.rs"
```

Comment:
Without changing this the `cargo mutants` command would not see the content of the files and call the mutations for them